subprojects/gradle/impact/src/main/kotlin/com/avito/impact/plugin/ImpactMetricsSender.kt

```diff
@@ -0,0 +1,28 @@
+package com.avito.impact.plugin
+
+import com.avito.android.sentry.EnvironmentInfo
+import com.avito.android.stats.GaugeMetric
+import com.avito.android.stats.StatsDSender
+import com.avito.android.stats.graphiteSeries
+import com.avito.impact.ConfigurationType
+
+class ImpactMetricsSender(
+    private val statsDSender: StatsDSender,
+    private val environmentInfo: EnvironmentInfo
+) {
+
+    private val prefix by lazy {
+        val envName = environmentInfo.environment.publicName
+        val node = environmentInfo.node?.take(32) ?: "_"
+        val buildId = environmentInfo.teamcityBuildId() ?: "nobuildid"
```

Comment:
```suggestion
        val buildId = environmentInfo.teamcityBuildId() ?: "_"
```
We frequently use this shortcut in metrics.